src/Components/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Display from './Display';
+import Button from './Button';
+import ButtonPanel from './ButtonPanel';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="app"> { this.props.children }
+        <Display />
+        <ButtonPanel>
+        <Button label="AC" value="clear" />
+          <Button label="7" value="7" />
+          <Button label="4" value="4" />
+          <Button label="1" value="1" />
+          <Button label="0" value="0" />
```

Comment:
- You need to make your App component to be functional instead of class-based, remember we are not setting state yet at this point. Please do make this a functional component. 
![image](https://user-images.githubusercontent.com/46115725/89368652-ad12f180-d6a1-11ea-9865-36b53aea5158.png)